src/core/sys/darwin/netinet/in_.d

```diff
@@ -0,0 +1,605 @@
+
+    enum IPV6_RECVPKTINFO = 61;
+
+    enum IPV6_RECVHOPLIMIT = 37;
+    enum IPV6_RECVRTHDR    = 38;
+    enum IPV6_RECVHOPOPTS  = 39;
+    enum IPV6_RECVDSTOPTS  = 40;
+
+    enum IPV6_USE_MIN_MTU  = 42;
+    enum IPV6_RECVPATHMTU  = 43;
+
+    enum IPV6_PATHMTU = 44;
+
+    enum IPV6_PKTINFO  = 46;
+    enum IPV6_HOPLIMIT = 47;
+    enum IPV6_NEXTHOP  = 48;
+    enum IPV6_HOPOPTS  = 49;
+    enum IPV6_DSTOPTS  = 50;
+    enum IPV6_RTHDR    = 51;
+
+    enum IPV6_AUTOFLOWLABEL = 59;
+
+    enum IPV6_DONTFRAG = 62;
+
+    enum IPV6_PREFER_TEMPADDR = 63;
+
+    enum IPV6_BOUND_IF = 125;
+
+    enum IPV6_RTHDR_LOOSE  = 0;
+    enum IPV6_RTHDR_STRICT = 1;
+    enum IPV6_RTHDR_TYPE_0 = 0;
+
+    enum IPV6_DEFAULT_MULTICAST_HOPS = 1;
+    enum IPV6_DEFAULT_MULTICAST_LOOP = 1;
+
+    enum IPV6_MIN_MEMBERSHIPS = 31;
+    enum IPV6_MAX_MEMBERSHIPS = 4095;
+
+    enum IPV6_MAX_GROUP_SRC_FILTER = 512;
+    enum IPV6_MAX_SOCK_SRC_FILTER  = 128;
+
+    struct in6_pktinfo
+    {
+        in6_addr ipi6_addr;
+        uint     ipi6_ifindex;
+    };
+
+    struct ip6_mtuinfo
+    {
+        sockaddr_in6 ip6m_addr;
+        uint         ip6m_mtu;
+    };
+
+    enum IPV6_PORTRANGE_DEFAULT = 0;
+    enum IPV6_PORTRANGE_HIGH    = 1;
+    enum IPV6_PORTRANGE_LOW     = 2;
+
+    enum IPV6PROTO_MAXID = IPPROTO_PIM + 1;
+
+    enum IPV6CTL_FORWARDING         = 1;
+    enum IPV6CTL_SENDREDIRECTS      = 2;
+    enum IPV6CTL_DEFHLIM            = 3;
+    enum IPV6CTL_DEFMTU             = 4;
+    enum IPV6CTL_FORWSRCRT          = 5;
+    enum IPV6CTL_STATS              = 6;
+    enum IPV6CTL_MRTSTATS           = 7;
+    enum IPV6CTL_MRTPROTO           = 8;
+    enum IPV6CTL_MAXFRAGPACKETS     = 9;
+    enum IPV6CTL_SOURCECHECK        = 10;
+    enum IPV6CTL_SOURCECHECK_LOGINT = 11;
+    enum IPV6CTL_ACCEPT_RTADV       = 12;
+
+    enum IPV6CTL_LOG_INTERVAL   = 14;
+    enum IPV6CTL_HDRNESTLIMIT   = 15;
+    enum IPV6CTL_DAD_COUNT      = 16;
+    enum IPV6CTL_AUTO_FLOWLABEL = 17;
+    enum IPV6CTL_DEFMCASTHLIM   = 18;
+    enum IPV6CTL_GIF_HLIM       = 19;
+    enum IPV6CTL_KAME_VERSION   = 20;
+    enum IPV6CTL_USE_DEPRECATED = 21;
+    enum IPV6CTL_RR_PRUNE       = 22;
+    enum IPV6CTL_V6ONLY         = 24;
+    enum IPV6CTL_RTEXPIRE       = 25;
+    enum IPV6CTL_RTMINEXPIRE    = 26;
+    enum IPV6CTL_RTMAXCACHE     = 27;
+
+    enum IPV6CTL_USETEMPADDR     = 32;
+    enum IPV6CTL_TEMPPLTIME      = 33;
+    enum IPV6CTL_TEMPVLTIME      = 34;
+    enum IPV6CTL_AUTO_LINKLOCAL  = 35;
+    enum IPV6CTL_RIP6STATS       = 36;
+    enum IPV6CTL_PREFER_TEMPADDR = 37;
+    enum IPV6CTL_ADDRCTLPOLICY   = 38;
+    enum IPV6CTL_USE_DEFAULTZONE = 39;
+
+    enum IPV6CTL_MAXFRAGS   = 41;
+    enum IPV6CTL_MCAST_PMTU = 44;
+
+    enum IPV6CTL_NEIGHBORGCTHRESH      = 46;
+    enum IPV6CTL_MAXIFPREFIXES         = 47;
+    enum IPV6CTL_MAXIFDEFROUTERS       = 48;
+    enum IPV6CTL_MAXDYNROUTES          = 49;
+    enum ICMPV6CTL_ND6_ONLINKNSRFC4861 = 50;
+
+    enum IPV6CTL_MAXID = 51;
+
```

Comment:
Per the linux version of in.h, they come from an obsolete RFC, so I didn't put them in either the FreeBSD or OS X bindings.